docs/AboutUs.md

```diff
@@ -11,51 +11,47 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Calvin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lordidiot.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/lordidiot)]
+[[portfolio](team/lordidiot.md)]
 
-* Role: Project Advisor
+- Role: Team lead
```

Comment:
Please add `Scheduling and tracking` as well

Reply:
Have rectified the issues, please review again.